SU2_CFD/src/solver_adjoint_mean_inc.cpp

```diff
@@ -3490,51 +3489,11 @@ void CAdjIncEulerSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_contain
       case MASS_FLOW_RATE:
         Psi_outlet[0]+=1;
         break;
-      case OUTFLOW_GENERALIZED:
```

Comment:
I noticed this is in the incompressible adjoint solver. did these outlet options work within that solver too? Perhaps we need to trim down more?